src/Psalm/Storage/ClassLikeStorage.php

```diff
@@ -242,6 +242,10 @@ class ClassLikeStorage
     public $appearing_method_ids = [];
 
     /**
+     * Map from lowercase method name to list of declarations in order from parent, to grandparent, to
+     * great-grandparent, etc **including traits and interfaces**. Ancestors that don't have their own declaration are
+     * skipped.
+     *
```

Comment:
This seems to be accurate for the cases I looked at, but I haven't checked in depth to make sure.